src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelMetrics.cs

```diff
@@ -138,6 +146,8 @@ private void ConnectionStopCore(in ConnectionMetricsContext metricsContext, Exce
 
     public void ConnectionRejected(in ConnectionMetricsContext metricsContext)
```

Comment:
Nit, now that it's a class, do we want the `in` modifier? There are a bunch of other instances that could be changed too.

```suggestion
    public void ConnectionRejected(ConnectionMetricsContext metricsContext)
```